designer/server/src/main/scala/pl/touk/nussknacker/ui/component/ComponentService.scala

```diff
@@ -136,7 +134,7 @@ class DefaultComponentService(
     componentsDefinition.toList
       .map { case (info, definition) =>
         // TODO: We should add componentId into ComponentDefinitionWithImplementation, thx to that we won't need to use ComponentIdProvider in many places
-        val componentId = componentIdProvider.createComponentId(processingType, info)
+        val componentId = definition.componentIdUnsafe
```

Comment:
let's remove usages of comoponentIdProvider in places where we replaced them by componentIdUnsafe